spec/rubocop/cop/rails/active_record_aliases_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe RuboCop::Cop::Rails::ActiveRecordAliases do
+  subject(:cop) { described_class.new }
+
+  describe '#update_attributes' do
+    before do
+      inspect_source('book.update_attributes(author: "Alice")')
+    end
+
+    it 'registers an offense' do
+      expect(cop.offenses.size).to eq 1
+      expect(cop.highlights).to eq ['update_attributes']
+      expect(cop.messages).to eq(
+        ['Prefer update over update_attributes.']
+      )
+    end
+  end
+
+  describe '#update_attributes!' do
+    before do
+      inspect_source('book.update_attributes!(author: "Bob")')
+    end
+
+    it 'registers an offense' do
+      expect(cop.offenses.size).to eq 1
+      expect(cop.highlights).to eq ['update_attributes!']
+      expect(cop.messages).to eq(
+        ['Prefer update! over update_attributes!.']
+      )
```

Comment:
Perhaps it is better to use `expect_offense` method.
https://github.com/bbatsov/rubocop/blob/v0.52.1/spec/rubocop/cop/rails/active_support_aliases_spec.rb#L10-L15

It matches the template in case we make a new cop.
https://github.com/bbatsov/rubocop/blob/v0.52.1/lib/rubocop/cop/generator.rb#L55-L77

Also, writing a no offense and autocorrect test cases will increase confidence in the code and maintainability.